Marlin/language_de.h

```diff
@@ -36,12 +36,17 @@
 #define WELCOME_MSG                         MACHINE_NAME " bereit"
 #define MSG_SD_INSERTED                     "SD-Karte erkannt"
 #define MSG_SD_REMOVED                      "SD-Karte entfernt"
+#define MSG_LCD_ENDSTOPS                    "Endstopp" // Max length 8 characters
 #define MSG_MAIN                            "Hauptmenü"
 #define MSG_AUTOSTART                       "Autostart"
 #define MSG_DISABLE_STEPPERS                "Motoren aus" // M84
```

Comment:
"Motoren deaktivieren" is better here imho. "Motoren aus" is more suited for a status indication rather than the actual act of disabling the steppers.